common/scripts/configs/system_codes.sql

```diff
@@ -775,9 +775,14 @@ do $$
       values (504, 'account', 'viewType', '54188262bc4d591ba438d62a', '54188262bc4d591ba438d62a', '2016-06-01', '2016-06-01');
     end if;
 
-    if not exists (select 1 from "systemCodes" where code = 505) then
+    if not exists (select 1 from "systemCodes" where code = 506) then
       insert into "systemCodes" ("code", "name", "group", "createdBy", "updatedBy", "createdAt", "updatedAt")
-      values (505, 'showNotBuilt', 'viewObjectType', '54188262bc4d591ba438d62a', '54188262bc4d591ba438d62a', '2018-06-25', '2018-06-25');
+      values (506, 'hideNotBuilt', 'viewObjectType', '54188262bc4d591ba438d62a', '54188262bc4d591ba438d62a', '2018-07-17', '2018-07-17');
+    end if;
+
+    -- Delete showNotBuilt systemCode
+    if exists (select 1 from information_schema.columns where table_name = 'systemCodes') then
```

Comment:
Is it safe to delete this? Won't there be any references to this code?

Reply:
we are not using this anywhere.

Reply:
I have added it before to add a option for show/hide not built branches. But we have decided to rename it to `hideNotBuilt` instead of `showNotBuilt`

Reply:
Is `showNotBuilt` already added in production?

Reply:
If it not we should just clean it up in RC and use 505.

Reply:
yes its already added in production

Reply:
Okay, can we follow the below template for deleting the system codes? Looks good otherwise.

Reply:
okay